inspirehep/modules/theme/jinja2filters.py

```diff
@@ -613,18 +619,19 @@ def publication_info(record):
                         name="conference_only",
                         template="inspirehep_theme/format/record/Conference_info_macros.tpl",
                         ctx=ctx)
-                except TypeError:
-                    pass
             elif 'parent_recid' in pub_info and \
-                    'conference_recid' not in pub_info:
-                pid = PersistentIdentifier.get(
-                    'conferences', str(pub_info['parent_recid']))
-                parent_rec = es.get_source(
-                    index='records-hep',
-                    id=str(pid.object_uuid),
-                    doc_type='hep',
-                    ignore=404)
+                    'conference_recid' not in pub_info:                
                 try:
+                    pid = PersistentIdentifier.get(
+                        'conferences', str(pub_info['parent_recid']))
```

Comment:
The `parent_recid` is probably not in `conferences`. It might be in `literature`, for the case I tested: https://github.com/jacquerie/inspire-next/commit/007fe31b441faf88a5593489a6f46e341ea522c4#diff-cc4ec36bfdca525994bc68c9082219e3R951